packages/esm-appointments-app/src/hooks/patientAppointmentContext.ts

```diff
@@ -0,0 +1,13 @@
+import { createContext } from 'react';
+
+/**
+ * When rendering the Patient Appointments view, let the view know if this is being rendered in the Patient Chart or the Appointments App
+ * (we need to know this to determine if we open the appointments form as part of the patient chart workspace or as an overlay in the appointments app)
+ */
+export enum PatientAppointmentContextTypes {
```

Comment:
See comments above for explanation of use of this context

Reply:
I'm not totally clear on how this context is supposed to get set, but right now context doesn't get preserved across extension boundaries (this would change with [framework-native mounting](https://openmrs.atlassian.net/browse/O3-1418) but I'd prefer it if we didn't rely on that for this kind of thing). Assuming this is an extension you're working on. In which case, you might pass this info in as extension state.

Hopefully once we have a unified core workspace system we can do away with the split altogether—it will just "launch a workspace," whatever that means in the current context.